src/model/priors/flat.rs

```diff
@@ -155,10 +155,10 @@ impl PairModel<ContinuousAlleleFreqs, DiscreteAlleleFreqs> for FlatTumorNormalMo
                 likelihood_tumor(af_tumor, Some(af_normal))
             };
 
-            let p_tumor = if af_tumor.start == af_tumor.end {
-                density(*af_tumor.start)
+            let p_tumor = if af_tumor.inner.start == af_tumor.inner.end {
```

Comment:
All these `*.inner.*` changes will become superfluous with the Deref trait implementation shown above.